src/api/java/com/minecolonies/api/colony/buildings/ModBuildings.java

```diff
@@ -36,6 +36,7 @@
     public static final String        WAREHOUSE_ID      = "WareHouse";
     public static final String        POSTBOX_ID        = "Postbox";
     public static final String        FLORIST_ID        = "Florist";
+    public static final String        ENCHANTER_ID      = "enchanter";
```

Comment:
Shouldn't this be with a capital E?

Reply:
no, on purpose not, for compat with 1.14 =D (less work to port)